src/rgw/support/rgw_support_bundle

```diff
@@ -50,7 +50,7 @@ class RGWSupportBundle:
         """ Generate a tar file. """
         Log.info(f"{COMPONENT_NAME} support bundle generation started!!")
         coredumps = filters.get('coredumps', False)
-        stacktrace = filters.get('stacktrace', False)
+        stacktrace = True  # always include stacktrace else use filters.get('stacktrace', False)
```

Comment:
So stack trace generated by Motr clients alone, like RGW?
@nkommuri Motr service doesnt generate stack trace?

Reply:
got it